site/docs/patterns/dashboards/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This page opens when clicking the menu, which against what [navigation pattern](https://www.saltdesignsystem.com/salt/patterns/navigation#multi-level-group) suggests

Reply:
Checked with @lilyvc, Mosaic's side bar is built using third party sidebar and there's this limitations where parent item will navigate as well as expand the group. This behaviour will hopefully be gone when we switch to using Salt's accordion component